src/model/commands/isi.rs

```diff
@@ -9,6 +31,61 @@ pub struct Command {
     pub payload: Vec<u8>,
 }
 
+impl Command {
+    pub fn apply(&self, world_state_view: &mut WorldStateView) {
+        println!("World state: {:?}", world_state_view.world);
+        match self.command_type {
+            1 => {
```

Comment:
It's better to use enum with explicit tag in my opinion here, if we have to use raw integer values.

Reply:
How to deal with custom ISI?

Reply:
Added a link to RH2-2 to think about this case. Resolving for now.